RELEASES.md

```diff
@@ -1,3 +1,78 @@
+Version 1.29.0 (2018-09-13)
+==========================
+
+Compiler
+--------
+- [Bumped minimum LLVM version to 5.0.][51899]
+- [Added `powerpc64le-unknown-linux-musl` target.][51619]
+- [Added `aarch64-unknown-hermit` and `x86_64-unknown-hermit` targets.][52861]
+
+Libraries
+---------
+- [`Once::call_once` now no longer requires `Once` to be `'static`.][52239]
```

Comment:
I'm not sure what this means. Are you referring to the closure being passed to `call_once`? `Once` is either `'static 'or not, it's not a generic type -- the user has no control over whether that type is `'static`

Reply:
Not the closure, previously `call_once` takes a `&'static self`.

Reply:
What do you think about rephrasing this as "`Once::call_once` now takes `&self` instead of `&'static self`"?